programs/keeper/Keeper.cpp

```diff
@@ -52,6 +58,14 @@ int mainEntryClickHouseKeeper(int argc, char ** argv)
     }
 }
 
+#ifdef KEEPER_STANDALONE_BUILD
+
+void collectCrashLog(
+    Int32, UInt64, const String &, const StackTrace &)
+{}
```

Comment:
It seems Darwin doesn't like weak symbols, the standalone build failed on my Mac.

Reply:
Maybe leave a comment in the code about it?